.github/workflows/linux.yml

```diff
@@ -66,7 +92,7 @@ jobs:
 
   # Build libamrex and all tutorials w/o MPI
   tutorials-nonmpi:
-    name: GNU@7.5 C++14 non-MPI [tutorials]
+    name: GNU@7.5 C++14 NOMPI [tutorials]
```

Comment:
@WeiqunZhang please mark this as "not required" until the PR is merged, since I change the name of that test.